chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/auth/UserDTO.java

```diff
@@ -230,20 +240,12 @@ public void setPasswordResetRequired(final boolean passwordResetRequired) {
 
     @Override
     public String toString() {
-        String ret = "[UserDTO: "
-                + "[id: " + this.id + "]"
-                + "[subjectName: " + this.subjectName + "]"
-                + "[fullName: " + this.fullName + "]"
-                + "[friendlyName: " + this.friendlyName + "]"
-                + "[email: " + this.email + "]"
-                + "[phoneNumber: " + this.phoneNumber + "]"
-                + "[title: " + this.title + "]"
-                + "[signatureDate: " + this.signatureDate + "]"
-                + "[failedLoginCount: " + this.failedLoginCount + "]"
-                + "[accountExpired: " + this.accountExpired + "]"
-                + "[accountLocked: " + this.accountLocked + "]"
-                + "[credentialsExpired: " + this.credentialsExpired + "]"
-                + "[accountEnabled: " + this.accountEnabled + "]"
+        String ret = "[UserDTO: " + "[id: " + this.id + "]" + "[subjectName: " + this.subjectName + "]" + "[fullName: "
```

Comment:
I think this must be done from some eclipse auto reformatting? But it doesn't look like it makes the code any easier to read... probably less so. Same for all of the queries in the UserDAOImpl class... whatever Eclipse thinks is proper formatting for the HQL strings does not improve the readability to us developers. Is there an easy way to undo that formatting?